src/Esfa.Vacancy.Register.Api/Controllers/GetApprenticeshipVacancyController.cs

```diff
@@ -44,7 +46,7 @@ public GetApprenticeshipVacancyController(GetApprenticeshipVacancyOrchestrator a
         /// </summary>
         [HttpGet]
         [AllowAnonymous]
-        [Route("{vacancyReference:int}")]
+        [Route("{vacancyReference:int}", Order = 0)]
```

Comment:
I'm a little confused about this. 😕 
Instead of 404 from the action below it seems we would want  400 instead. Understand where you are coming from regarding the handling of routes.

Reply:
Also, if we were to have more than one route endpoint in this controller, we can specify AllowAnonymous on the controller

Reply:
Single endpoint accepting string will resolve this